src/cli.rs

```diff
@@ -49,6 +49,9 @@ pub struct VersionCommand {
     /// Bump to a patch release version, regardless of the conventional commits
     #[clap(long)]
     pub patch: bool,
+    /// Only commits that update those files/folders will be taken into account. It is useful to support monorepos.
+    #[clap(short = 'F', long = "file")]
```

Comment:
as f and files was not used yet this can just be:

```suggestion
    #[clap(short, long)]
```

maybe files could be renamed to paths, as file can either be a folder or a file

Reply:
I've renamed files to paths, but now I need to specify a short name because -p is used as the prefix.